front/app/modules/commercial/content_builder/admin/components/CraftComponents/Container/index.tsx

```diff
@@ -20,7 +21,9 @@ Container.craft = {
   rules: {
     canMoveIn: (nodes) =>
       nodes.every(
-        (node) => node.data.type === Text || node.data.type === Image
+        ((node) => node.data.type === AboutBox) ||
+          ((node) => node.data.type === Text) ||
+          ((node) => node.data.type === Image)
       ),
```

Comment:
There was a bug here where the rules were flaky and sometimes components were not permitted in a container when they should be. Using this format fixes the bug though.

Reply:
Ah, interesting. But alright 👍